src/rust/src/x509/verify.rs

```diff
@@ -0,0 +1,29 @@
+// This file is dual licensed under the terms of the Apache License, Version
+// 2.0, and the BSD License. See the LICENSE file in the root of this repository
+// for complete details.
+
+use pyo3::IntoPy;
+
+use crate::x509::certificate::Certificate as PyCertificate;
+
+#[pyo3::pyclass(name = "Store", module = "cryptography.hazmat.bindings._rust.x509")]
+struct PyStore(pyo3::Py<pyo3::types::PyList>);
+
+#[pyo3::pymethods]
+impl PyStore {
```

Comment:
NB: I intentionally avoided adding a `#[getter]` for the underlying certificate list here, under the presumption that we don't want to lock ourselves into stores only working with/on certificates (since the RFCs are clear that a "trust root" is really just a subject + key that might _happen_ to be in certificate form).